src/playwright/types.ts

```diff
@@ -0,0 +1 @@
+export type Timeout = { timeout?: number };
```

Comment:
I used this in 1 of my suggestions:
```
type LocatorOptions = {
  timeout?: number;
  delayMs?: number;
};
```

Reply:
Didn't take this into use as I just made use of the Timeout type in 88a56d7c0fd58126bbbfac90e1decdc47130e441 and split the delaying into its own function.